### ISSUE TITLE
[Accessibility] The 'Clear all (x)' button in the combo box is not accessible by keyboard.

### ISSUE THREAD

**Issue by Rich43** [ng-select/ng-select#2100]
**Describe the bug**
The 'Clear all (x)' button in the combo box is not accessible by keyboard.

**Reproducbile example**
[Example](https://stackblitz.com) which reproduces described behaviour

**To Reproduce** 
Steps to reproduce the behavior (if example is not provided):
1. Go to ng select component.
2. Select options and the clear all button on the right display.
3. Tried to use "Tab" button to focus the clear all while can not focus by keyboard.

**Expected behavior**
When the clear all button display, can use the keyboard to focus and do the operation.

**Screenshots**
![image](https://user-images.githubusercontent.com/527504/173567917-e3c19c8e-1da1-4e6a-a71f-5ee355cd06cb.png)

**Desktop (please complete the following information):**
- OS: Windows 10
- Browser Chrome
- Angular Version 12.2.13
- Ng Select Version 7.4.0

**Comment by Vadym-Kyrylenko** [comment]
this issue remained unsolved
